### PR TITLE
Add VCS options for `workspaces create` subcommand

### PR DESCRIPTION
**Pull request by jakewan**
This feature is to support connecting a Terraform Cloud workspace to an existing GitHub repo/branch upon creation.

### CODE REVIEW

File: cmd/workspaces_create.go [hunk 1]
```diff
@@ -55,15 +64,40 @@ func (c *workspacesCreateCmd) Run() error {
...
+	w, err := c.deps.client.workspaces.create(
+		client,
+		context.Background(),
+		c.OrgOpts.name,
+		opts,
+	)
 	if err != nil {
 		return err
 	}
-	if workspace == nil {
```
Comment (LyleScott) [inline cmd/workspaces_create.go:65]: I personally favor `workspace` over `w` since it's more meaningful and leaves less to assumption.

Comment (jakewan) [reply cmd/workspaces_create.go:65]: So do I generally. But there's a preference in the Go community for short, even one letter variable names, so I do this when writing Go.

**Review by LyleScott — APPROVED** [review]
Seems ok and straightforward logically. I'm just not a fan of single character var names outside of a few common things. Take `w` for example; Sometimes it means writer, sometimes it means workspace. Some might say the locality of that variable being used might seem "obvious" since initialization/usage of the variable are so close (which i don't buy...). If that's allowed, then one inevitably starts stringing them together in the case of `return c.r.Run()` which just leads to non readable code IMHO.

File: cmd/workspaces_create.go (hunk 1 above)
Comment (jakewan) [reply cmd/workspaces_create.go:65]: Having said that, I don't mind changing this.

**Pull request merged**